### DISCUSSION TITLE
Future idea: Bonding curve, curation marketing, article competition

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#30] [Ideas]
These features are not for the epoch 2.1. They are planned for epoch 3.0

## Changes on existing features:

- The contest will include TApp token valuations as a component of your wallet assets. Your investment on TApps will be calculated based on the TApps' token current sell price. So miners have an additional investment choice besides only reinvesting to TEA mining.

## New added features:
- Create a new TApp, with initial investment funded by the owner.
- Curators can invest in TApps. They pay TEA and receive the TApps' token via the bonding curve as an automated market maker (AMM).
- Curators can sell any TApp's tokens at any time to the TApp's bonding curve AMM.

## Award distribution:
- $1500 award for miners based on the valuation of net asset value in their wallet. There is no change from epoch 2.1.
- Additional $1500 awarded to the single best TApp. It will be distributed to all the TApp's token holders.

## How to define "the best TApp"?
We will choose a social channel and pick a relevant criteria.

- If we decide to use YouTube videos as the competitive field for our TApps, the total views is the only number we'll compare.
- If we choose articles (on the Medium blog for example), then reads will be counted.
- If we use Twitter as the chosen network for our epoch 3.0 competition, then we'll count retweets.

We will decide which media we will use before it starts, but there will be only one winner. If we finally decide to have two, then an additional $1500 will be added to this new award.

## Creating TApp: Suppose we use Medium post in epoch 3.0
- The core team will post several existing medium posts, and record the initial read count.
- Based on the team posted blogs, anyone can write their own article related to TEA and post to Medium as well.
- The creator of the new article will automatically receive the  initial bonus of his own TApp's token. Also, he'll have the privilege to invest the first funds to receive the locked TApp token. The bonus TApp tokens are locked and cannot be sold. The bonus tokens will help the creator if they have no TEA to invest at the beginning.

## What could curators do to win? 

Both creators (and curators) are incentivized to create a good article (or find one) and get more people to read it. The higher the read count, the more chance there is to win the $1500 award. 

If curators find a good article and decide it will be the winner, they can invest in it and use their social influence to share and bump it in multiple social media channels. This will bring more readers that will read the article and bump the read count.

If a curator later finds out an article probably won't win, he can sell this TApp token and reinvest it in another article. Of course, selling tokens early will likely be at a loss.

## Notes and tips:

- TApp doesn't have to mean application. An article, a YouTube video could be also considered a TApp. Anything that has value and will bring in increased revenue in the future can be considered a TApp.
- Google search **Bonding Curve** and **Curation Marketing** if you're not familiar with those terms.
- Picking a good TApp to invest in means that you'll not only share in the $1500 award of the article, but you'll also increase the value of assets in your wallet by skillfully investing your TEA in promising TApps. The contest rules are the same as in the original 2.1 epoch mining contest: those who have the most assets valued in USD at the end of the contest are the winners of the $1500 prize. In epoch 3.0, we've just added the value of the TApp tokens in your wallet as part of the overall asset valuation.
- Bonding curves have an inherent feature where tokens are cheaper at the beginning of the curve and become higher in price further along the curve. Investing early would be the best strategy. Once you've invested, try to get others to join. They'll help you pump the TApp's token price.

======================================================
Please join the discussion of this idea. Once we've developed this feature, we may start testing even before epoch 2.1 ends. Everyone in this discussion will be invited to join the testing and the next epoch.

**Comment by puchungotv** [comment]
Youtube should be a better medium to have a define definition of what the project is and also gives people a visual of the wallet and how to mine. 

Where can i read more about: Curators can invest in TApps. They pay TEA and receive TApps' token via the Bonding Curve as AMM.

I am a little confused about this process.

**Reply by kevingzhang** [reply]
Youtube is one of our top choices. My only concern is making videos is harder. If we choose youtube video, how many video creators are in our community? 

This video explains bonding curve very well https://youtu.be/cUQYVDhEYkM

Spend TEA to buy TApp'stoken? I hope you can get the answer from this video

**Comment by geopor** [comment]
It's probably better if contestants can focus on being curators. I recommend videos because technical topics can be dry when written, and videos are more naturally engaging.

The team can decide what content is shareable and promotable to help publicize the content. I don't think we have to pick one platform over another. As a matter of fact, using more than one will help drive more engagement across different social channels.

## Example
The team decides to promote 3 YouTube videos. Each YouTube video also has a Tweet promoting it and a Medium post with graphics and transcript.

Each video is promotable through:
- YouTube directly (liking)
- Twitter (likes)
- Medium (handclaps)

Using similar engagement metrics makes comparing easier among platforms. Using this cross platform format also lets contestants promote the post on the platform where they are the most comfortable or have the biggest influence. Medium is a smaller platform so many contestants might not have an account, but being multi-platform lets them vote and promote wherever they have an account.

The winning YouTube video is the one that has the highest likes (YouTube and Twitter) or handclaps (Medium). Whatever is the largest # on whichever platform is declared the winner.

**Reply by kevingzhang** [reply]
Please correct me if I understood incorrectly. You propose to have 3 videos. All the contestants are all curators, they can invest to one (or more) of the 3 videos. They can try their best to share or cross promotion. The highest like video win.

If we mix the twitter, medium and youtube, how do you decide a 1000 view youtube vs 10000 like twitter, which one is the winner?

**Reply by geopor** [reply]
We don't use views (or "reads" in Medium), we use likes (or "handclaps" in Medium).

Let's look at an example:

1. Ken Jee publishes a YouTube video: https://www.youtube.com/watch?v=uXLnbdHMf8w
2. He sends a tweet promoting the video: https://twitter.com/kenjee_ds/status/1299734773456203777
3. He creates an article from the video and includes a link back to the original YouTube video using Medium: https://towardsdatascience.com/why-im-starting-data-science-over-21bec8036ce9
(this is a custom domain but he is using Medium as the platform)

Let's imagine that this video was a TApp on the contest - how would we evaluate it against other TApps of the other YouTube videos?

1. We see how many likes it got on YouTube:

<img width="704" alt="Screen Shot 2021-08-18 at 10 23 09 PM" src="https://user-images.githubusercontent.com/86096370/130012882-b72c1513-cfc5-402b-8f2c-76125eb3a370.png">

2. We check the number of likes it got on Twitter:

<img width="605" alt="Screen Shot 2021-08-18 at 10 25 04 PM" src="https://user-images.githubusercontent.com/86096370/130012886-a44be888-3608-48d3-a858-7d8e41365db4.png">

3. And we check the number of "handclaps" the relevant article got on Medium:

<img width="738" alt="Screen Shot 2021-08-18 at 10 25 58 PM" src="https://user-images.githubusercontent.com/86096370/130012887-5a967c6f-13e0-4e33-aac2-bf412026866f.png">

The formula for determining the winner could be:
1. Add up all the likes/handclaps to get the number. For this example, it would be 2,500 + 263 + 292 = **3,055**
2. Another possible way to pick the winner is to use whatever is the greatest number from all the platforms, which in this case would be YouTube at **2,500**.

**Comment by geopor** [comment]
We can open a discussion for ideas on videos, and people can add some cross promotion ideas. I would recommend keeping the reward at 1500 and using some extra funds, maybe 100-200 per video, for paying for cross promotion to other YouTube channels. Promotion is very important, and if someone is so-so on a particular platform but likes the cross promotion idea they may support that video in the contest. It also allows us to test out some marketing techniques.
